.github/workflows/devworkspace-generator-release.yml

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2020-2021 Red Hat, Inc.
+# Copyright (c) 2023 Red Hat, Inc.
```

Comment:
Why not 2020-2023, just wondering

Reply:
I suppose copyright years go like A or A-B, where A is year of first publication, and B is year of latest modification